server.js

```diff
@@ -32,6 +36,12 @@ app.get("/users/:id/responders", middleware.checkToken, user.getResponders);
 app.get("/users/:id/responders/count",middleware.checkToken,user.getResponderCount);
 app.delete("/users/:id/responders", middleware.checkToken, user.deleteResponders);
 
+// Help requests
+app.post("/help-requests", middleware.checkToken, help_request.addHelpRequest);
```

Comment:
Hold on shouldn't it be
```suggestion
app.post("/help-request", middleware.checkToken, help_request.addHelpRequest);
```
?

Reply:
Nope, this is fine, its appending one help request to a collection of help requests.

Reply:
readme needs to be changed then. also in all your pull requests you wrote “/help-request”, which is why i pointed it out

Reply:
Oh must have been a typo, my bad!